pyproject.toml

```diff
@@ -77,6 +77,12 @@ hf = [
   "numba>=0.60.0",
   "datasets[audio,vision]>=2.21.0"
 ]
+video = [
+  "av<14",
```

Comment:
Q: why `<14`?

Reply:
On v14 it is an error: `AttributeError: 'av.video.codeccontext.VideoCodecContext' object has no attribute 'close'`, thread: https://github.com/PyAV-Org/PyAV/discussions/1700

Reply:
is it temporary? can we put a link here to it as a comment then please?

Reply:
Sure! Good comment, thank you 👍